src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -111,11 +111,13 @@ public function getEnv($prefix, $name, \Closure $getEnv)
         }
 
         if ('const' === $prefix) {
+            $env = \str_replace("\\\\","\\", $env);
```

Comment:
this should be removed (the blog post has a typo, the `\` should not be doubled there, we're going to fix it)

Reply:
We've tested here with and without the double \ but we could not get it to work.

The original scripts runs:                        return constant($name);
While I only could get it to work with:     return constante($env);

Will this be fixted to? Or are we doing something wrong?